Makefile

```diff
@@ -5,6 +5,7 @@ VERSION := v0.1.0
 
 .PHONY: proto
 proto:
+	find -name "*.pb.go" -delete
```

Comment:
Please add the deleting action to the `clean` target. Maybe you can replace the all the individual `rm -rf xxx.pd.go` by the `find -delete`.